src/scancodes.rs

```diff
@@ -372,8 +373,7 @@ impl ScancodeSet for ScancodeSet2 {
             0x59 => Ok(KeyCode::ShiftRight),         // 59
             0x5A => Ok(KeyCode::Enter),              // 5A
             0x5B => Ok(KeyCode::BracketSquareRight), // 5B
-            0x5D => Ok(KeyCode::HashTilde),          // 5D
-            0x61 => Ok(KeyCode::BackSlash),          // 61
+            0x5D => Ok(KeyCode::BackSlash),          // 5D
```

Comment:
OK, so I have BackSlash (which is typically labelled `\ |`) and HashTilde (which is `# ~` on a UK keyboard, but more correctly called 'Europe 2') backwards. Rather than remove HashTilde can you add it back in as 0x61?

Reference: This table from microsoft https://download.microsoft.com/download/1/6/1/161ba512-40e2-4cc9-843a-923143f3456c/translate.pdf

Reply:
The Microsoft document is actually the first one that mentions the 0x61 scancode. The references I based my research on were https://www.win.tue.nl/~aeb/linux/kbd/scancodes-10.html and https://wiki.osdev.org/PS/2_Keyboard#Scan_Code_Set_2; both of which do not mention them. I'll put the 0x61 scancode with the HashTilde keycode back.